src/map/skill.c

```diff
@@ -5317,23 +5337,25 @@ int skill_castend_nodamage_id(struct block_list *src, struct block_list *bl, uin
 				x1 = sd->bl.x + range;
 				y1 = sd->bl.y + range;
 
-				elemental->delete(sd->ed,0);
-
-				if(!skill->check_unit_range(src,src->x,src->y,skill_id,skill_lv))
-					ret = skill->castend_pos2(src,src->x,src->y,skill_id,skill_lv,tick,flag);
-				for(i = 0; i < MAX_PARTY; i++) {
-					struct map_session_data *psd = p->data[i].sd;
-					if(!psd)
-						continue;
-					if(psd->bl.m != sd->bl.m || !psd->bl.prev)
-						continue;
-					if(range && (psd->bl.x < x0 || psd->bl.y < y0 ||
-						psd->bl.x > x1 || psd->bl.y > y1))
-						continue;
-					if(!skill->check_unit_range(bl,psd->bl.x,psd->bl.y,skill_id,skill_lv))
-						ret |= skill->castend_pos2(bl,psd->bl.x,psd->bl.y,skill_id,skill_lv,tick,flag);
+				clif->skill_nodamage(src,bl,skill_id,skill_lv,1);
+ 				elemental->delete(sd->ed,0);
+				skill->unitsetting(src,MG_SAFETYWALL,skill_lv + 5,src->x,src->y,0);
+				skill->unitsetting(src,AL_PNEUMA,1,src->x,src->y,0);
```

Comment:
Wasn't this the bugged behavior on release? As far as I know the skill only casts SW now.